frontend/www/js/problem.edit.form.js

```diff
@@ -13,31 +13,36 @@ omegaup.OmegaUp.on('ready', function() {
         $('#problem-form .tag-list').append(
           $('<a></a>')
             .attr('href', '#tags')
+            .attr('data-key', e.name)
```

Comment:
nit:

```suggestion
            .data('key', e.name)
```

por consistencia